src/fcs-fable/src/Compiler/Service/FSharpCheckerResults.fs

```diff
@@ -3908,7 +3912,12 @@ type FSharpCheckProjectResults
                     | _ -> TcSymbolUses.Empty)
                 |> Array.toSeq
 #endif //!FABLE_COMPILER
-            | Choice2Of2 tcSymbolUses -> Async.RunSynchronously(tcSymbolUses, ?cancellationToken = cancellationToken)
+            | Choice2Of2 tcSymbolUses ->
+#if FABLE_COMPILER
+                seq {}
+#else
+                Async.RunSynchronously(tcSymbolUses, ?cancellationToken = cancellationToken)
+#endif //!FABLE_COMPILER
```

Comment:
@ncave 

To improve the user experience and follow the same pattern as other missing replacement, I remove support for `Async.RunSynchronously` at compile time.

This surfaces the issues that `fcs-fable` had some call against `Async.RunSynchronously`.

Do we need to port these changes somewhere else too? I believe this part of the code was never hit when running the compilation or Fable REPL otherwise we would have had a runtime exception reported.

Reply:
@MangelMaxime Yes, they need to be ported [here](https://github.com/ncave/fsharp/pull/1), but I can do that.

Reply:
My assumption was it is not used because otherwise we would get an exception as the implementation was:

```ts
export function runSynchronously(): never {
  throw new Error("Asynchronous code cannot be run synchronously in JS");
}
```

Reply:
Yes, what I meant was someone *could* use it, as it's client-facing API.
The REPL clearly doesn't use it, so your change is good for now.